tests/Fixtures/TestBundle/Entity/FooDummy.php

```diff
@@ -35,11 +37,21 @@ class FooDummy
     #[ORM\Id]
     #[ORM\GeneratedValue(strategy: 'AUTO')]
     private ?int $id = null;
+
     /**
      * @var string The foo name
      */
     #[ORM\Column]
     private $name;
+
+    #[ApiProperty(writable: false)]
+    #[ORM\Column(nullable: true)]
+    public $nonWritableProp;
```

Comment:
actually instead of setting `writable: false` explicitly we could provide only a getter without setter?